src/pages/Landing/components/Body/ChatBox/Input/Input.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react'
+import happyIcon from 'assets/images/happy.png'
+import attachIcon from 'assets/images/attach.png'
+import microIcon from 'assets/images/micro.png'
+import styles from './Input.module.scss'
+
+export default () => (
+  <div className={styles.Input}>
+    <img src={happyIcon} alt="happy" />
+    <img src={attachIcon} alt="attach" />
+    <input type="text" placeholder="Type messages here..." />
+    <img src={microIcon} alt="micro" />
+  </div>
```

Comment:
reusable componnent

Reply:
Which component needs to reusable? please clarify it